### ISSUE TITLE
VSCodeInsider_DefaultSettingforNewEnvironment_ColorContrast:Color contrast ratio is less than 4.5 for texts present in the screen.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/vscode#89095]
**Environment Details:​**
VSCode Version: 1.41.0-insider (System setup)
Operating System: MacOS
Mac version: 10.15.1 (19B88)

**Repro Steps:​**
1. Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Online"
3. Click on "Visual Studio Online" and navigate till "Remote Explorer" ,click on it.
4. Navigate till "+" button beside the  "VS ONLINE" control and click on it .
5. Select any existing environment and then again click on the "+" button .
6. Open color contrast analyzer
7. Check color contrast for the text present in the screen.

**Actual:**
For the texts, The color contrast ratio is less than 4.5.

**Expected:​**
For the texts, The color contrast ratio should be >= 4.5.

**Same issue Observed for the below scenario:**
**1.VS ONLINE-Select Plan**
1.Open VS Code insider 
2.Navigate till "Extension" button and search for the "Visual Studio Online"
3.Click on "Visual Studio Online" and navigate till "Remote Explorer" ,click on it.
4.Navigate till "Select Plan" button beside the  "VS ONLINE" control and click on it.

**​User Impact:​**
Low vision user does not understand/unable the saw the  control if it is not  visible clearly.
![image](https://user-images.githubusercontent.com/55909580/72899338-cf6ba000-3d4b-11ea-9f64-153fcf78941a.png)

**Comment by isidorn** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/88752
